src/controllers/expenses.contr.js

```diff
@@ -0,0 +1,68 @@
+const expensesServ = require('../services/expenses.service');
```

Comment:
Why don't you use a full name expensesService. 

I think now it looks a bit weird